Sources/SwiftKuery/Table.swift

```diff
@@ -201,48 +364,30 @@ open class Table: Buildable {
         return true
     }
     
-    /// Add a foreign key to the table.
-    ///
+    /**
+     Function to set a single `Column` instances, as a composite foreign key, in the `Table` instance.
```

Comment:
Please reword description as above examples for primary key. Note there are plurals in the current description which should be singular.